### PR TITLE
[pages] Update Jekyll guide instructions to use RUBY_VERSION

### PR DESCRIPTION
**Pull request by kristianfreeman**

### CODE REVIEW

**Comment by KimJ15** [comment]
@codewithkristian @WalshyDev Is this PR still something we want to consider?

File: content/pages/framework-guides/deploy-a-jekyll-site.md [hunk 1]
```diff
@@ -82,6 +82,16 @@ Deploy your site to Pages by logging in to the [Cloudflare dashboard](https://da
 
 </div>
 
+Add an [environment variable](/pages/platform/build-configuration/#environment-variables) that matches the Ruby version that you are using locally. Set this as `RUBY_VERSION` on both your preview and production deployments. Below, `3.1.3` is used as an example:
```
Comment (WalshyDev) [inline content/pages/framework-guides/deploy-a-jekyll-site.md]:
```suggestion
Add an [environment variable](/pages/platform/language-support-and-tools/) that matches the Ruby version that you are using locally. Set this as `RUBY_VERSION` on both your preview and production deployments. Below, `3.1.3` is used as an example:
```

File: content/pages/framework-guides/deploy-a-jekyll-site.md [hunk 2]
```diff
@@ -82,6 +82,16 @@ Deploy your site to Pages by logging in to the [Cloudflare dashboard](https://da
...
+
+<div>
+
+| Environment variable | Value          |
+| -------------------- | -------------- |
+| `RUBY_VERSION`       | `3.1.3`        |
+
+</div>
```
Comment (WalshyDev) [inline content/pages/framework-guides/deploy-a-jekyll-site.md]:
```suggestion
| Environment variable | Value          |
| -------------------- | -------------- |
| `RUBY_VERSION`       | `3.1.3`        |
```

**Review by WalshyDev — APPROVED** [review]
Happy with this and looks good to me

cc: @deadlypants1973

**Review by KimJ15 — APPROVED** [review]

**Comment by KimJ15** [comment]
@codewithkristian you can merge when ready

File: content/pages/framework-guides/deploy-a-jekyll-site.md [hunk 3]
```diff
@@ -17,10 +17,10 @@ If you have an existing Jekyll site on GitHub Pages, refer to [the Jekyll migrat
 
 Jekyll is written in Ruby, meaning that you will need a functioning Ruby installation, like `rbenv`, to install Jekyll.
 
-To install Ruby on your computer, follow the [`rbenv` installation instructions](https://github.com/rbenv/rbenv#installation) and select a recent version of Ruby by running the `rbenv` command in your terminal:
+To install Ruby on your computer, follow the [`rbenv` installation instructions](https://github.com/rbenv/rbenv#installation) and select a recent version of Ruby by running the `rbenv` command in your terminal. The Ruby version you install will also be used to configure the Pages deployment for your application.
 
 ```sh
-$ rbenv install 2.7.2
+$ rbenv install $RUBY_VERSION # For instance, 3.1.3
```
Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-a-jekyll-site.md]:
```suggestion
$ rbenv install <RUBY_VERSION> # For example, 3.1.3
```

**Review by deadlypants1973 — APPROVED** [review]

**Comment by deadlypants1973** [comment]
heyo @codewithkristian I missed that this was in draft, but ping me if you need anything on ot

**Comment by kodster28** [comment]
Merging in b/c this has several approvals (and confirmed with Walshy that it's fine).

**Pull request merged**